src/onnxruntime_benchmark/include/utils.hpp

```diff
@@ -21,9 +21,9 @@ enum class DataPrecision : unsigned int {
     FP32 = 0,
     FP16,
     U8,
-    S8,
-    S32,
-    S64,
+    I8,
```

Comment:
From my point of view pair U.. and S.. was better (U8 and S8). More clear after using OpenCV with CV_32**S**, CV_8**U**, CV_32**F**.

Reply:
Yes, maybe, I changed mostly because of onnx dtype namings:
```
    {ONNX_TENSOR_ELEMENT_DATA_TYPE_INT8, DataPrecision::I8},
    {ONNX_TENSOR_ELEMENT_DATA_TYPE_INT32, DataPrecision::I32},
    {ONNX_TENSOR_ELEMENT_DATA_TYPE_INT64, DataPrecision::I64},
```